test/test_memoryview.py

```diff
@@ -2,11 +2,134 @@
 # coding: utf-8
 
 
+from array import array
 from msgpack import packb, unpackb
+import sys
 
 
-def test_pack_memoryview():
-    data = bytearray(range(256))
-    view = memoryview(data)
-    unpacked = unpackb(packb(view))
-    assert data == unpacked
+# For Python < 3:
+#  - array type only supports old buffer interface
+#  - array.frombytes is not available, must use deprecated array.fromstring
+if sys.version_info[0] < 3:
+    def __memoryview(obj):
```

Comment:
Just out of habit. If something is not intended to be used elsewhere, I make it private. I made a ``make_memoryview`` function instead now.
However, I'm not really satisfied with this solution myself. Old-style buffers don't have type information, so this does not test float packing in 2.7 at all. So far if have only been able to get properly typed memoryviews from numpy.